params/protocol_params.go

```diff
@@ -193,6 +193,10 @@ const (
 )
 
 var (
+	// Dummy Randao fields to be used in a Randao-enabled Genesis block.
+	ZeroRandomReveal = make([]byte, 96)
+	ZeroMixHash      = make([]byte, 32)
```

Comment:
Rather `Zero`, `Empty` seems more fit.

Reply:
I was trying to blend in with "ZeroBaseFee", which is replacement to header.BaseFee before fork.
Similarly, ZeroMixHash is replacement to header.MixHash before fork.